packages/review/src/components/full-list-button.tsx

```diff
@@ -45,7 +45,7 @@ export const FullListButton = ({
   const { getWindowId } = useTripleClientActions()
 
   const reviewListUrl = `/reviews/list?_triple_no_navbar&${qs.stringify({
-    region_id: regionId,
+    ...(regionId && regionId !== null && { region_id: regionId }),
```

Comment:
```suggestion
    region_id: regionId ?? undefined,
```

qs.stringify가 undefined를 받으면 omit 해주니 더 간략하게 작성하면 좋겠습니다.

Reply:
![Uploading image.png…]()

Reply:
regionId가 null string으로 넘어오는 케이스가 존재하고 있는데,  qs.stringify가 null string을 정상적인 값으로 인지해버려서 위와 같이 처리를 할수밖에 없다고 판단했습니다.

Reply:
스레드 보고 왔는데 regionId 값이 string으로 `'null'` 인거에요? 그러면 
```suggestion
    ...(regionId && regionId !== 'null' && { region_id: regionId }),
```
이렇게 해야되지 않나요?

Reply:
'null'로 체크해야겠네여! 리뷰 감사합니다! [1476c97](https://github.com/titicacadev/triple-frontend/pull/3266/commits/1476c97a2aa8cd6eb3590315d347496954436041)